### ISSUE TITLE
Qwen3-VL 8B Instruct - Large amount of time spent acquiring locks

### ISSUE THREAD

**Issue by najarvis** [QwenLM/Qwen3-VL#2004]
For me the inference time of Qwen3-VL 8B is very slow, and when I used cProfile to inspect where the model is taking the longest, I found that the vast majority of the time spent is simply on acquiring thread locks. When I run the same code but using Llama-3.2-11B the time spent here is trivial.

```
980271 function calls (898365 primitive calls) in 215.536 seconds

   Ordered by: internal time

   ncalls  tottime  percall  cumtime  percall filename:lineno(function)
   250/16  203.162    0.813    0.000    0.000 {method 'acquire' of '_thread.lock' objects}
     3407    3.422    0.001    3.422    0.001 {method 'to' of 'torch._C.TensorBase' objects}
        1    3.074    3.074    3.074    3.074 {built-in method torch.conv3d}
        6    0.968    0.161    4.548    0.758 _contextlib.py:117(decorate_context)
       31    0.916    0.030    0.916    0.030 {method 'read' of '_ssl._SSLSocket' objects}
       36    0.858    0.024    0.858    0.024 {built-in method torch.nonzero}
  ...
```

I have not been able to determine where the thread locks are taking place

```python

    
class Qwen3Model(ModelHelper):

    ckpt = "Qwen/Qwen3-VL-8B-Instruct"
    model_name = "Qwen3-VL"

    MAX_PIXELS = 384 * 384
    MIN_PIXELS = 224 * 224

    def __init__(self, device='cpu'):
        self._model = Qwen3VLForConditionalGeneration.from_pretrained(
            Qwen3Model.ckpt,
            dtype=torch.bfloat16,
            device_map=device,
            attn_implementation="flash_attention_2",
        )

        self._processor = AutoProcessor.from_pretrained(Qwen3Model.ckpt)
        self._device = device

    ...
    def compute_label_logits_batch(self, image_paths: list[str], base_prompt: str, emotions: list[str]):
        # image_logits = []

        # WARNING: Does not handle when multiple emotions start with the same token. Does not work with the EMOTIC dataset yet.
        emotion_ids = torch.tensor(list(self._processor.tokenizer.encode(" " + emo)[0] for emo in emotions)).to(self._device)
        
        messages = []
        for image in image_paths:
            # For each image, we want to find the probability the next token generated corresponds to each emotion
            messages.append([
                {
                    "role":"user",
                    "content":[
                        {
                            "type": "image",
                            "url": image
                        },
                        {
                            "type":"text",
                            "text": f"Describe this image using one of the following {len(emotions)} emotions: {", ".join(emotions)}"
                        }
                    ]
                },
                {
                    "role": "assistant", 
                    "content": [
                        {
                            "type": "text",
                            "text": base_prompt 
                        }
                    ]
                }
            ])

        inputs = self._processor.apply_chat_template(
            messages,
            tokenize=True,
            return_dict=True,
            return_tensors="pt",
            continue_final_message=True,
            padding=True,
            max_pixels=Qwen3Model.MAX_PIXELS,
        )
        inputs.pop("token_type_ids", None)
        inputs = inputs.to(self._device)

        with torch.no_grad():
            outputs = self._model(**inputs)
    ...
```

I don't believe I'm doing anything out of the ordinary, as I based the code on the examples in the GitHub and Hugging Face pages. I am using the model's forward method as opposed to generate because I need the logits for the next tokens. I know that because of the padding this won't actually be exactly what I'm looking for, but that isn't relevant to the performance concern.

Thanks

```
torch==2.9.1
transformers==4.57.6
flash_attn==2.8.3

**Comment by joshua840** [comment]
I encountered the same issue and hope that no one else will face it in the future. The problem is that cuDNN does not support Conv3D in PyTorch 2.9.x. Downgrading to PyTorch 2.8.0 fixed the problem for me, as also noted in #1811.

**Comment by najarvis** [comment]
Gah thank you. Now I feel silly for spending so much time on this 🤦‍♂️ I see it is fixed in torch 2.10 which should be releasing soon so hopefully this should be fully resolved in the next few days. Thank you!
